src/core/utils/useReservableFromAnotherLibrary.tsx

```diff
@@ -0,0 +1,28 @@
+import { useConfig } from "./config";
+import { getAllFaustIds } from "./helpers/general";
+import { useGetHoldings } from "../../apps/material/helper";
+import { Manifestation } from "./types/entities";
+
+const useReservableFromAnotherLibrary = (manifestations: Manifestation[]) => {
```

Comment:
Great with the hook.
I think tou should add some tests testing the hook now that you have deleted the old tests.